cloudferry/discover.py

```diff
@@ -176,6 +177,7 @@ def discover_all(cfg, cloud):
         LOG.debug('Finished discovering %s using %s',
```

Comment:
Can we add comment that: " Results of discovery can be found in migration_data.db" ?